pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -13,6 +13,9 @@ import (
 // For example: `master` or `worker`
 const MachineConfigRoleLabelKey = "machineconfiguration.openshift.io/role"
 
+// KubeletConfigRoleLabelPrefix is the label that must be present in the KubeletConfig CR
+const KubeletConfigRoleLabelPrefix = "pools.operator.machineconfiguration.openshift.io/"
```

Comment:
Based on the following example(https://github.com/openshift/machine-config-operator/blob/master/docs/KubeletConfigDesign.md#example---setting-the-kubelet-log-level), I understood that the `pools.operator.machineconfiguration.openshift.io/{worker|master}: ""` label must be present in the KubeletConfig CR inorder to get picked up by the respective pool `worker|master`. Hence I have defined a constant here for the prefix so that it can be used accordingly.

Reply:
gah sorry, this is fine, it needs to be used by controller manager/apiserver/etc...